index.html

```diff
@@ -3,8 +3,36 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
     <title>Document</title>
+    <link rel="styles" href="css/styles.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
```

Comment:
usually when we have a pull request, we avoid having code that we dont need :) 
for this question: please create the classs when you style it, not in advance.

this will help you keep your code clean.